### PR TITLE
Bug process documentation

### PR DESCRIPTION
**Pull request by RobinWagner**
Adding bug process documentation to docs

### CODE REVIEW

**Comment by RobinWagner** [comment]
Hi @ulferts,
I added the bug process documentation that we wrote to our docs website.
May I ask you to review this when you have a moment?
Thank you,
Robin

**Comment by birthe** [comment]
HI @HDinger und @RobinWagner, can we merge this to our docs? We can deploy it on Edge and then add more information to it or do more finetuning.

File: docs/development/bug-process/README.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+
+
+
+### 4. Assigning bugs to version & fixing bugs
+
+<u>Involved teams:</u>
+
+- Software Development
+
+<u>Summary</u>:
```
Comment (HDinger) [inline docs/development/bug-process/README.md:130]: That whole block has changed in the mean time. With our continuous bug fixing process they way we work on the prioritised bugs is different.

**Review by HDinger — CHANGES_REQUESTED** [review]
The part about how we fix the bugs is unfortunately outdate by now. So I'd say we cannot merge it like this. I can work it over, when I have time but it will only be after the release.

**Comment by HDinger** [comment]
Shall this be published with version 12.2 or is it more something we would like to have internally? If the former, we'd need to change the base branch to `release/12.2`.

**Comment by cbliard** [comment]
This doc helped me to understand the Bug process when I joined OpenProject. I was surprised to find it in pull requests (Jens pointed me to it). It was January 2022 back then.

I understand that the document is incomplete because it is missing the continuous bug fixing process, but for newcomers I think it still have value. Publishing this content somewhere would give it more visibility.

**Comment by oliverguenther** [comment]
A different process is now followed and documented here: https://community.openproject.org/projects/development/wiki/query-based-bugfixing

**Pull request closed without merging**